1st draft at HowTo (#239)
oscar-system/Oscar.jl: WhatIs.md

# Oscar - what is the idea

> Oscar should be competing with (and beating) Magma and Sage in our areas of

Reviewer: ```suggestion
The ultimate goal for Oscar should be to compete with (and ideally beat) Magma and Sage in our areas of
```

expertise.

Oscar is the innovative, next generation Computer Algebra System

Oscar is written in Julia, but is not Julia, nor can it be. 

>  Example:

Reviewer: ```suggestion
 Some examples to illustrate what that means:
```

>   Julias matrices are (c)-Arrays (of arbitrary dimension), parametrized by

Reviewer: ```suggestion
  Julias matrices are (C) arrays (of arbitrary dimension), parametrized by
```
